weblogo/seq_io/clustal_io.py

```diff
@@ -131,7 +137,7 @@ def read(fin, alphabet=None):
 #     space and then the sequence.
 
 
-def _scan(fin):
+def _scan(fin: TextIO) -> Iterator[Token]:
```

Comment:
Return type doesn't look right. Should be Generator?

Reply:
Actually probably ok. The return statement at 207 makes this look weird. The return isn't needed, remove.

Reply:
remove the return or change return to `-> None` ?

Reply:
Remove the return statement on line 207. Not needed I think.

Reply:
ooh! got it. Thanks!